tests/test_elasticache.py

```diff
@@ -75,6 +75,33 @@ def test_elasticache_cluster_simple_filter(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_elasticache_snapshot_copy_cluster_tags(self):
+        session_factory = self.replay_flight_data(
+            'test_elasticache_copy_cluster_tags')
+
+        snapshot_name = 'test-tags'
+
+        results = session_factory().client('elasticache').list_tags_for_resource(ResourceName='arn:aws:')['TagList']
```

Comment:
before running the policy, but after loading it, policy.manager.generate_arn  might help.